core/eval.go

```diff
@@ -616,6 +616,35 @@ func evalEXPIRETIME(args []string) []byte {
 	return Encode(int(exTimeMili/1000), false)
 }
 
+// evalEXPIREAT sets a expiry time(in unix-time-seconds) on the specified key in args
+// args should contain 2 values, key and the expiry time to be set for the key
+// The expiry time should be in integer format; if not, it returns encoded error response
+// Returns RespOne if expiry was set on the key successfully.
+// Once the time is lapsed, the key will be deleted automatically
+func evalEXPIREAT(args []string) []byte {
+	if len(args) <= 1 {
```

Comment:
From redis docs,

> The EXPIREAT command supports a set of options:
NX -- Set expiry only when the key has no expiry
XX -- Set expiry only when the key has an existing expiry
GT -- Set expiry only when the new expiry is greater than current one
LT -- Set expiry only when the new expiry is less than current one
A non-volatile key is treated as an infinite TTL for the purpose of GT and LT. The GT, LT and NX options are mutually exclusive.

We can add support for these commands in a separate PR, could you help with creating an issue for the same?

Reply:
sure I will create an issue for both `EXPIRE` and `EXPIREAT` commands